### ISSUE TITLE
Ci/Cd quickstart giving errors 

### ISSUE THREAD

**Issue by jlpl1999** [aws-quickstart/quickstart-trek10-serverless-enterprise-cicd#9]
Hello QuickStart:
I have down load all your github yaml and trying to plug in appropriate parameters for our AWS Account. 
Another **important** things that I removed all reference **KMS** from across yaml files.

**Reason**: with current IAM role in our organization, I can not create KMS key, therefore, i must removed that from the cf scripts

I have contact AWS Rep and they suggested to open issues here, and let your team investigate further with any suggestion using your base line template without KMS featurs.

Thank you and please see all yaml file attached that I use for this lab
[quickstart-no-kms.zip](https://github.com/aws-quickstart/quickstart-trek10-serverless-enterprise-cicd/files/3266539/quickstart-no-kms.zip)

**Comment by jayyeras** [comment]
Thank you for your interest in the Quick Start and sharing your experience. While we cannot support modified versions of the QuickStart, I’ll tag this as a feature request to see if we can provide an option to disable the use of kms in a future release. If you do manage to get this working feel free to send in a pull request.

**Comment by jlpl1999** [comment]
@jayyeras Yes thank you and I will keep following this issues with your new tag features.

**Comment by awarzon** [comment]
Thanks for the suggestion. I like the idea of adding a parameter like this but see one problem @jayyeras and @jlpl1999 . 

There are two places that KMS is used:
* `ArtifactBucketKey`: We can change to SSE-S3 encryption if "UseKMS" is false... this is no problem.
*  `SecretsManagerKey`: This is a problem because of the secrets manager limitation [described here](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-secretsmanager-secret.html#cfn-secretsmanager-secret-kmskeyid)

> You can use the account's default CMK to encrypt and decrypt only if you call this operation using credentials from the same account that owns the secret. If the secret is in a different account, then you must create a custom CMK and specify the ARN in this field.

Any ideas of what we would do in this case? Drop secrets manager altogether if "UseKMS" is false?
